packages/react/src/themes/teams/components/Dropdown/dropdownItemStyles.ts

```diff
@@ -3,12 +3,22 @@ import { DropdownVariables } from './dropdownVariables'
 import { DropdownItemProps } from '../../../../components/Dropdown/DropdownItem'
 
 const dropdownItemStyles: ComponentSlotStylesInput<DropdownItemProps, DropdownVariables> = {
-  root: ({ props: p, variables: v }): ICSSInJSStyle => ({
+  root: ({ props: p, variables: v, theme: { siteVariables } }): ICSSInJSStyle => ({
     whiteSpace: 'nowrap',
+    border: `1px solid transparent`,
     backgroundColor: v.listItemBackgroundColor,
+    ...(p.selected && {
+      fontWeight: v.listItemSelectedFontWeight,
+      color: v.listItemSelectedColor,
+    }),
     ...(p.active && {
-      color: v.listItemColorActive,
-      backgroundColor: v.listItemBackgroundColorActive,
+      ...(p.isFromKeyboard && {
```

Comment:
Does the new focus solution @Bugaa92 setup in #1269 not work here?

Reply:
Unfortunately not, will sync today again with @Bugaa92 on this.

Reply:
This is how the new focus solution looks: 
![image](https://user-images.githubusercontent.com/4512430/57614579-e7164600-7579-11e9-9326-ac474d0eb025.png)
More important that it is breaking the accessibility (on keys up and down, this border is not moving to the right list element). @Bugaa92 FYI

Reply:
Manage to fix this, the `position: 'relative'` is necessary for the element where the border focus styles are applied. Should we add this somehow in the border focus styles?